docs/troubleshooting.rst

```diff
@@ -49,6 +49,15 @@ Poetry provides different packages according to the folder, and depends
 on the ``pyproject.toml`` file in the current folder. Make sure to run
 ``poetry`` in the root folder of LISA.
 
+Missing a lot of packages
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, make sure you are currently in the root of your code folder (where you
+cloned LISA source code). Then run ``poetry install`` again to ensure that
+poetry has successfully set up all the dependencies. If you still have problems
+after running this command, refer to `Issue #1401
```

Comment:
1. It doesn't need to refer to the Issue.
2. This FAQ is similar with "Cannot find package after run ``poetry install``". We can update the title to "Missing a lot of packages". It's clearer, because it's part of the error message.
3. The original one doesn't suggest to run `poetry install` again. It should be added.